ros2doctor/README.md

```diff
@@ -7,6 +7,8 @@ It will examine your ROS 2 setup, such as distribution, platform, network interf
 
 ## Usage
 
+Windows and OSX users: run `python3 -m pip install ifcfg` to enable network check.
```

Comment:
I don't believe `python3` will generically work on Windows. The executable is called `python` in my experience, but probably depends on how the user installed python. Our [install documentation](https://index.ros.org/doc/ros2/Installation/Dashing/Windows-Install-Binary/#install-dependencies) uses `python` for Windows. Speaking of which, I think we could list this install instruction on the ROS 2 installation documentation instead of here. What do you think?

Reply:
Good point.